### PR TITLE
added checkbox component doc

### PR DESCRIPTION
**Pull request by RahmanC**

### CODE REVIEW

File: content/docs/components/checkbox/usage.mdx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+id: Checkbox
+title: Checkbox
+package: '@nature-ui/checkbox'
+image: 'components/checkbox.svg'
+description:
+  Checkbox component is used in forms when a user needs to select multiple
+  values from several options.
+---
+
+The `Checkbox` component is used in forms when a user needs to select multiple
```
Comment (DNature) [inline content/docs/components/checkbox/usage.mdx:11]: This looks like a duplicate of the description. I'd suggest you override the description with this sentence

File: content/docs/components/checkbox/usage.mdx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+```
+
+### Checkbox with custom color
+
+You can override the color of the `Checkbox` to any color key specified in
+`theme.colors`.
+
+```jsx
+<Stack spacing={10} row>
+  <Checkbox color='purple-500' defaultIsChecked>
```
Comment (DNature) [inline content/docs/components/checkbox/usage.mdx:50]: Color prop is deprecated. please see the [storybook documentation](https://github.com/Nature-UI/nature-ui/blob/6ec4a1de81fd3963c02aafefdc9f476d1678f071/packages/checkbox/stories/checkbox.stories.tsx) for more details. 

we now use: `className="bg-purple-500"`

**Review by DNature — CHANGES_REQUESTED** [review]
Please always compare your changes with the storybook docs.

File: content/docs/components/checkbox/usage.mdx (hunk 1 above)
Comment (RahmanC) [reply content/docs/components/checkbox/usage.mdx]: This has been resolved

File: content/docs/components/checkbox/usage.mdx (hunk 2 above)
Comment (RahmanC) [reply content/docs/components/checkbox/usage.mdx]: this issue has been updated

**Comment by DNature** [comment]
Hi @RahmanC , please click the "Resolve conversation" button once you've resolved any comments. Also ping me on discord so I can review your PR

File: content/docs/components/checkbox/usage.mdx [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+```
+
+### Checkbox with custom color
+
+You can override the color of the `Checkbox` to any color key specified in
+`theme.colors`.
+
+```jsx
+<Stack spacing={10} row>
+  <Checkbox color='bg-purple-500' defaultIsChecked>
```
Comment (DNature) [inline content/docs/components/checkbox/usage.mdx:48]: Replace `color` with `className`

File: content/docs/components/checkbox/usage.mdx [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+    Checkbox
+  </Checkbox>
+  <Checkbox color='bg-green-500' defaultIsChecked>
```
Comment (DNature) [inline content/docs/components/checkbox/usage.mdx:51]: Same here

File: content/docs/components/checkbox/usage.mdx [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+
+```jsx
+<Stack spacing={10} row>
+  <Checkbox size='sm' color='bg-purple-500'>
+    Checkbox
+  </Checkbox>
+  <Checkbox size='md' color='bg-green-500' defaultIsChecked>
+    Checkbox
+  </Checkbox>
+  <Checkbox size='lg' color='bg-orange-500' defaultIsChecked>
```
Comment (DNature) [inline content/docs/components/checkbox/usage.mdx:70]: Same here. `color` should be `className`

**Review by DNature — CHANGES_REQUESTED** [review]

**Comment by DNature** [comment]
@RahmanC This PR has been opened for too long. I'll close it by the end of this week so that someone else can pickup the ticket.

**Review by DNature — APPROVED** [review]

**Comment by DNature** [comment]
Screenshot 2023-03-31 at 20 25 38

**Pull request merged**